### DISCUSSION TITLE
Numpy 2.0

### DISCUSSION THREAD

**Discussion by NorihisaSegawa** [pf-robotics/kachaka-api#94] [Q&A]
皆様
pythonのdemoを動かそうとしたところ、Numpy を使っているものが動かなくなりました。
たぶん、Numpyが、メジャーアップデートしたからだと思われます。
対応法を教えてもらうと助かります。
よろしくお願いします。

**Comment by kmurase-pfr** [comment]
報告ありがとうございます。
以下でnumpyのメジャーアップデートに引っ張られないようにしました。
https://github.com/pf-robotics/kachaka-api/pull/96
